api/exchange.go

```diff
@@ -237,6 +237,19 @@ type ExchangeShim interface {
 	FillTrackable
 }
 
+// TradeMetricsHandler is invoked by the MetricsTracker to process new trades
+type TradeMetricsHandler interface {
```

Comment:
not sure I understand why this is an interface. I think this should just be a concrete struct implementing `api.FillHandler`